classes/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+class Game < Item
+  attr_accessor :title, :multiplayers, :lastplayed_date
+
+  def initialize(title, publish_date, archived, multiplayers, lastplayed_date)
+    super(nil, publish_date, archived)
+    @title = title
+    @multiplayers = multiplayers
+    @lastplayed_date = lastplayed_date
+  end
+
+  attr_writer :author
```

Comment:
it would be great if you can add the author as attr_accessor like this: `attr_accessor :publish_date, :author, :archived`